contributing.md

```diff
@@ -137,6 +138,15 @@ The gulpfile.js file contains the following targets that can be called manually
 | `copy-generated-client` |              | Copies the generated HTTP client code to dist                           |
 | `prepublish`            | `prepublish` | Runs the `clean` and `build:raw` tasks before publishing to npm         |
 
+## Debugger
+
+Debugging configurations are stored in launch.json file. Here're what they are and what they do:
```

Comment:
It's probably obvious once you're in that file, but consider adding an example for how to debug a specific command, showing how to add its arguments.